app/views/articles/_search_results.html.erb

```diff
@@ -17,7 +17,13 @@
         <div class="search_num_of_results">
           <div class='results-heading'>
             <h1 class="visually-hidden"><%= t('blacklight.search.page_heading', search_type: search_type_name) %></h1>
-            <h2><%= pluralize(number_with_delimiter(@response.response[:numFound]), "#{search_type_name} result") %></h2>
+            <h2>
+              About <%= pluralize(number_with_delimiter(@response.response[:numFound]), "#{search_type_name} result") %>
+              <span class="duplicates">Duplicates removed</span>
+              <button class="btn duplicate-info" aria-label="explanation" data-bs-container="body" data-bs-toggle="popover" data-bs-placement="right" data-bs-content="<%= t('.duplicates') %>">
```

Comment:
I am not sure explanation is a good aria-label

Reply:
What is your suggestion?

Reply:
That I am not sure. Maybe something like `results explanation`?

Reply:
It's more of an explanation of "Duplicates removed", not of the results.

Reply:
Isn't sort of both? the explanation has to do with the results `This may cause the number of displayed results to differ from the total found.` But it could also be Duplicates removed info? Honestly I don't really have a good handle on what a good aria label is but explanation seems a little too 🤷. The icon is an info icon so at the very least I think it should be `info`